string record_type
string equ_name
string ele_name
int8 equ_type
int8 ele_unique_id
